modules/apps/commerce/commerce-channel-web/src/main/resources/META-INF/resources/commerce_channel/general.jsp

```diff
@@ -149,17 +149,16 @@ Map<String, String> contextParams = HashMapBuilder.<String, String>put(
 
 				<div class="mb-4" id="autocomplete-root"></div>
 
-				<aui:select label="price-type" name="priceDisplayType">
-
-					<%
-					String priceDisplayType = commerceChannel.getPriceDisplayType();
-					%>
+				<%
+				String priceDisplayType = commerceChannel.getPriceDisplayType();
+				%>
 
+				<aui:select label="price-type" name="priceDisplayType" value="<%= priceDisplayType %>">
```

Comment:
I'm not sure this is right, I mean I cannot find any example of aui:select with the value attribute, I'll try this fix locally to be sure